packages/Algernon.package/ALGItem.class/instance/nameLessThan.given..st

```diff
@@ -4,6 +4,4 @@ nameLessThan: anItem given: aLowercaseString
 	| me other |
 	me := self lowerCaseName beginsWith: aLowercaseString.
 	other := anItem lowerCaseName beginsWith: aLowercaseString.
-	(me and: [other not]) ifTrue: [^ true].
-	(other and: [me not]) ifTrue: [^ false].
-	^ #equals
\ No newline at end of file
+	^(me and: [other not]).
```

Comment:
This code is almost the same as `initalsLessThan:given:` please factor out the duplication.